Assets/02_Scripts/UI/LevelButton.cs

```diff
@@ -36,9 +28,17 @@ public void AddStars()
         UpdateStars();
     }
 
+    public void UpdateLevel()
+    {
+        var button = this.GetRequiredComponent<Button>();
+        var unlockedLevels = PlayerPrefs.GetInt("UnlockedLevels", 0);
+        button.interactable = unlockedLevels >= LevelIndex;
```

Comment:
`LevelIndex` ist beim ersten Spielstart immer `0`.
Schau mal in `LevelSelector:30` rein, da wird der Index erst später als `OnEnable` gesetzt.